Documentation/Types/TtContent.rst

```diff
@@ -0,0 +1,106 @@
+
+..  note::
+
+    The fields are added to the :confval:`types-showitem` through their corresponding
+    :ref:`palettes <palettes>`. In case such palette has been changed
+    by extensions, the required system fields are added individually to corresponding tabs.
+
+In case one of those palettes has been changed to no longer
+include the corresponding system fields, those fields are added individually
+depending on their definition in the :ref:`ctrl` section.
+
+By default, all custom fields - the ones still defined in :confval:`types-showitem` - are
+added after the `general` palette and are therefore added to the
+:guilabel:`General` tab, unless a custom tab (for example :guilabel:`Plugin`,
+or :guilabel:`Categories`) is defined in between. It's also possible to start
+with a custom tab by defining a `--div--` as the first item in the
+:confval:`types-showitem`. In this case, the :guilabel:`General` tab will be omitted.
+
+All those system fields, which are added based on the :php:`ctrl` section are
+also automatically removed from any custom palette and from the customized
+type's :confval:`types-showitem` definition.
+
+If the content element defines the :guilabel:`Extended` tab, it will be
+inserted at the end, including all fields added to the type via API methods,
+without specifying a position, via
+:php:`ExtensionManagementUtility::addToAllTcaTypes()`. See
+:ref:`types-content-examples-extended` for an example.
+
+..  _types-content-examples:
+
+Examples for the `showitems` TCA section in content elements
+============================================================
+
+..  _types-content-examples-basic:
+
+Basic custom content element with header and bodytext
+-----------------------------------------------------
+
+..  literalinclude:: _CodeSnippets/_basic_content_element.php
+
+The following tabs are shown, the header palette and bodytext field are shown
+in palette general:
+
+..  figure:: /Images/ManualScreenshots/tt_content_basic.png
+
+..  _types-content-examples-extended:
+
+Extended content element with custom fields
+-------------------------------------------
+
+..  literalinclude:: _CodeSnippets/_extended_content_element.php
+
+The following tabs are shown:
+
+..  figure:: /Images/ManualScreenshots/tt_content_extended.png
+
+Additional fields that are subsequently added to the end of the table using
+:php:`ExtensionManagementUtility::addToAllTcaTypes()` will appear in the tab
+:guilabel:`Extended`.
+
+..  _types-content-migration:
+
+Migration: Remove system fields from showitems on dropping TYPO3 v12.4 support
+==============================================================================
+
+It is sufficient to apply changes to the :confval:`types-showitem` section
+of content types once dropping TYPO3 v12.4 support in extensions.
+
+In site packages or extensions only supporting TYPO3 v13.3 or above you can
+migrate the :confval:`types-showitem` right away:
+
+..  literalinclude:: _CodeSnippets/tt_content_migration.diff
```

Comment:
Please add a caption for easy reception where to put this into.

And: is `:language: php` not missing?

Reply:
Caption is added, language is auto-detected